runtime/doc/editing.txt

```diff
@@ -1253,12 +1268,16 @@ directory.  Using a command to jump to another window doesn't change anything
 for the current directory.
 When a |:lcd| command has been used for a window, the specified directory
 becomes the current directory for that window.  Windows where the |:lcd|
-command has not been used stick to the global current directory.  When jumping
-to another window the current directory will become the last specified local
-current directory.  If none was specified, the global current directory is
-used.
-When a |:cd| command is used, the current window will lose his local current
-directory and will use the global current directory from now on.
+command has not been used stick to the tab- or global current directory.  When
+jumping to another window the current directory will become the last specified
+local current directory.  If none was specified, the tab- or global current
+directory is used.
+When changing tabs the same behaviour applies. If the current tab has no local
```

Comment:
Two spaces after dot. This is often forgotten, but Vim documentation separates sentenses by *two* spaces. You can observe this in the above text.